cnxpublishing/tests/views/test_admin.py

```diff
@@ -100,6 +101,81 @@ def test(self):
                 ]}, resp_data)
 
 
+class PrintStyleViewsTestCase(unittest.TestCase):
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        cls.settings = integration_test_settings()
+        from cnxpublishing.config import CONNECTION_STRING
+        cls.db_conn_str = cls.settings[CONNECTION_STRING]
+        cls.db_connect = staticmethod(db_connection_factory())
+
+    def setUp(self):
+        self.config = testing.setUp(settings=self.settings)
+        self.config.include('cnxpublishing.tasks')
+        init_db(self.db_conn_str, True)
+        with self.db_connect() as db_conn:
+            with db_conn.cursor() as cursor:
+                cursor.execute("""INSERT INTO files
+                                  (file, media_type) VALUES ('file', 'css');""")
+                cursor.execute("""INSERT INTO print_style_recipes
+                                  (print_style, fileid, tag)
+                                  VALUES ('ccap-physics', 1, '1.0');""")
+
+    def tearDown(self):
+        with self.db_connect() as db_conn:
+            with db_conn.cursor() as cursor:
+                cursor.execute("DROP SCHEMA public CASCADE")
+                cursor.execute("CREATE SCHEMA public")
+        testing.tearDown()
+
+    def test_print_styles(self):
+        request = testing.DummyRequest()
+
+        from ...views.admin import admin_print_styles
+        content = admin_print_styles(request)
+        self.assertEqual(1, len(content['styles']))
+        for row in content['styles']:
+            self.assertEqual(set(row.keys()),
+                             set(['print_style', 'file', 'type', 'revised',
+                                  'number', 'tag']))
+
+    def test_print_style_single(self):
+        request = testing.DummyRequest()
+        with self.db_connect() as db_conn:
+            with db_conn.cursor() as cursor:
+                cursor.execute("""INSERT INTO latest_modules
+                                  (print_style, portal_type, name, licenseid,
+                                        doctype, uuid, created, revised)
+                                  VALUES ('ccap-physics', 'Collection', 'test', 1,
+                                        'doc', 'aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa', now(), now());""")
+
+        print_style = 'ccap-physics'
+        request.matchdict['style'] = print_style
+
+        from ...views.admin import admin_print_styles_single
+        content = admin_print_styles_single(request)
+        self.assertEqual(content['print_style'], 'ccap-physics')
+        self.assertEqual(content['file'], 1)
+        self.assertEqual(content['recipe_type'], 'web')
+        for row in content['collections']:
+            self.assertEqual(set(row.keys()),
+                             set(['title', 'authors', 'revised', 'uuid',
+                                  'ident_hash', 'status']))
```

Comment:
How about we test for values as well? Maybe use one of the use-cases and adjust the data in the database to include a print-style.